core/src/trezor/ui/layouts/mercury/__init__.py

```diff
@@ -912,23 +931,21 @@ async def confirm_summary(
     br_type: str = "confirm_total",
     br_code: ButtonRequestType = ButtonRequestType.SignTx,
 ) -> None:
+    # TODO: info_title
     title = title or TR.words__title_summary  # def_arg
 
-    total_layout = RustLayout(
-        trezorui2.confirm_total(
-            title=title,
-            items=items,
-            info_button=bool(info_items),
-        )
-    )
-    info_items = info_items or []
-    info_layout = RustLayout(
-        trezorui2.show_info_with_cancel(
-            title=info_title if info_title else TR.words__title_information,
-            items=info_items,
+    await raise_if_not_confirmed(
+        RustLayout(
+            trezorui2.flow_confirm_summary(
+                title=title,
+                items=items or (),
+                fee_items=(),
+                account_items=info_items or (),
+            )
         )
+        # TODO br_type,
```

Comment:
remove comments since BR is sent from rust

Btw. in this case, the behavior would be the same even with `interact(..., br_type, br_code)` because the BR is sent only at the first screen of the flow, am I right? Whereas in `flow_confirm_output` we send multiple BRs along the flow.